src/MOI_wrapper.jl

```diff
@@ -31,6 +31,7 @@ const SUPPORTED_SCALAR_FUNCTIONS = Union{
 }
 
 const SUPPORTED_VECTOR_FUNCTIONS = Union{
+    MOI.VectorOfVariables,
     MOI.VectorAffineFunction{Float64}
```

Comment:
```suggestion
    MOI.VectorAffineFunction{Float64},
```

Reply:
does this also help?

Reply:
i guess i've to read a lot about Julia yet :sweat_smile: